packages/eas-cli/src/credentials/manager/ManageAndroid.ts

```diff
@@ -145,17 +145,17 @@ export class ManageAndroid {
   constructor(private callingAction: Action) {}
 
   async runAsync(ctx: Context, currentActions: ActionInfo[] = highLevelActions): Promise<void> {
+    const accountName = ctx.hasProjectContext
```

Comment:
Not sure if it's not done already, but if we don't prompt for every action maybe we should print relevant info e.g package name/bundle identifier so the users know which profile/app they are configuring?

Reply:
Yep, we do this already :)